config/settings.yaml.example

```diff
@@ -54,3 +63,5 @@
 #    :enabled: false
 #  :templates:
 #    :enabled: true
+#  :telemetry:
+#    :enabled: true
```

Comment:
Should this be? :

```rb
:loggers:
  :telemetry:
    :enabled: true
```

Also, here's some example output: https://gist.github.com/akofink/142c2dda7412b4fca8176fe5aa6d2c7e